### PR TITLE
Update perfetto to fix segmentation fault in trace processor

### PR DESCRIPTION
**Pull request by rschiu**
Bug: b/149739694

### CODE REVIEW

File: gapic/src/main/com/google/gapid/perfetto/models/CpuSummaryTrack.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
       "select quantum_ts, sum(dur)/cast(%d * %d as float) " +
       "from %s where utid != 0 group by quantum_ts";
   private static final String SLICE_RANGE_SQL =
-      "select row_id, ts, dur, cpu, utid, upid, end_state, priority " +
+      "select id, ts, dur, cpu, utid, upid, end_state, priority " +
```
Comment (stellama0208) [inline gapic/src/main/com/google/gapid/perfetto/models/CpuSummaryTrack.java]: I think it should be sched.id here, there would be "ambiguous column name: id" exception at area-selection in this panel. (This cpu summary panel appears when collapsing 8 cpus into one)/

File: gapic/src/main/com/google/gapid/perfetto/models/ProcessSummaryTrack.java [hunk 2]
```diff
@@ -38,13 +38,13 @@
...
+      "select quantum_ts, group_concat(id) ids, sum(dur)/cast(%d * %d as float) util " +
       "from %s group by quantum_ts";
-  private static final String SLICES_SQL = "select ts, dur, cpu, utid, row_id from %s";
+  private static final String SLICES_SQL = "select ts, dur, cpu, utid, id from %s";
   private static final String SLICE_RANGE_FOR_IDS_SQL =
-      "select row_id, ts, dur, cpu, utid, upid, end_state, priority " +
+      "select id, ts, dur, cpu, utid, upid, end_state, priority " +
       "from sched left join thread using(utid) " +
-      "where row_id in (%s)";
+      "where id in (%s)";
```
Comment (stellama0208) [inline gapic/src/main/com/google/gapid/perfetto/models/ProcessSummaryTrack.java]: Similarly believing sched.id here, same thing at click-selection in this panel.

File: gapic/src/main/com/google/gapid/perfetto/models/CpuSummaryTrack.java (hunk 1 above)
Comment (rschiu) [reply gapic/src/main/com/google/gapid/perfetto/models/CpuSummaryTrack.java]: Done

File: gapic/src/main/com/google/gapid/perfetto/models/ProcessSummaryTrack.java (hunk 2 above)
Comment (rschiu) [reply gapic/src/main/com/google/gapid/perfetto/models/ProcessSummaryTrack.java]: Done

**Comment by rschiu** [comment]
> Verified being able to open the trace. LGTM with two minor detail!

Thanks for catching the bugs!  PTAL

**Pull request merged**